assets/js/modules/adsense/components/common/AdSenseConnectCTA/index.js

```diff
@@ -124,89 +130,61 @@ export default function AdSenseConnectCTA() {
 
 	const handleDismissModule = useCallback( () => {
 		trackEvent( `${ viewContext }_adsense-cta-widget`, 'dismiss_widget' );
-		setDialogActive( true );
-	}, [ viewContext ] );
 
-	const handleConfirmDialog = useCallback( async () => {
-		await trackEvent(
-			`${ viewContext }_adsense-cta-widget`,
-			'dismiss_dialog_confirm'
+		// Check if the WordPress admin menu is open, and if not, open it.
+		const isAdminMenuOpen = !! document.querySelector(
+			'#wpwrap.wp-responsive-open'
 		);
-		await dismissItem( ADSENSE_CTA_WIDGET_DISMISSED_ITEM_KEY );
-	}, [ dismissItem, viewContext ] );
+		if ( ! isAdminMenuOpen ) {
+			document.getElementById( 'wp-admin-bar-menu-toggle' )?.click();
+		}
 
-	const handleDismissDialog = useCallback( () => {
-		trackEvent(
-			`${ viewContext }_adsense-cta-widget`,
-			'dismiss_dialog_cancel'
-		);
-		setDialogActive( false );
+		setShowTooltip( {
+			isTooltipVisible: true,
+			rehideAdminMenu: ! isAdminMenuOpen,
+		} );
 	}, [ viewContext ] );
 
-	useEffect( () => {
-		const handleDialogClose = ( e ) => {
-			// Close if Escape key is pressed.
-			if ( ESCAPE === e.keyCode ) {
-				setDialogActive( false );
-			}
-		};
+	const handleDismissTooltip = useCallback( async () => {
+		// If the WordPress admin menu was closed, re-close it.
+		if ( rehideAdminMenu ) {
+			document.getElementById( 'wp-admin-bar-menu-toggle' )?.click();
+		}
 
-		global.addEventListener( 'keyup', handleDialogClose );
+		await trackEvent(
+			`${ viewContext }_adsense-cta-widget`,
+			'dismiss_tooltip'
+		);
+		await dismissItem( ADSENSE_CTA_WIDGET_DISMISSED_ITEM_KEY );
+	}, [ dismissItem, rehideAdminMenu, viewContext ] );
 
-		return () => {
-			global.removeEventListener( 'keyup', handleDialogClose );
-		};
-	}, [] );
+	const cellProps = {
```

Comment:
@techanvil From a readability perspective, I would prefer if we inline those props because when looking at a specific `Cell`, we immediately know its size, instead of going back to the variable and check.

Reply:
@asvinb although I see your point to an extent, the problem with inlining the props here it is obscures the fact that the cells do in fact share these props, and it's necessary to compare them manually in order to notice that. To me, that makes it a bit less readable as it makes this higher level aspect, that the cells are "the same", harder to spot.